www/src/pages/migrating.mdx

```diff
@@ -1,145 +1,50 @@
-# Migrating to v1
+# Migrating to v2
 
-React Bootstrap v1 adds full compatibility with Bootstrap 4. Because bootstrap 4 is a major rewrite of the project there
-are significant breaking changes from the pre `v1` react-bootstrap.
+React Bootstrap v2 adds full compatibility with Bootstrap 5. Because bootstrap 5 is a major rewrite of the project there
+are significant breaking changes from v1 React Bootstrap.
```

Comment:
```suggestion
React-Bootstrap v2 adds full compatibility with Bootstrap 5. Because Bootstrap 5 is a major rewrite of the project, there
are significant breaking changes from React-Bootstrap v1.
```